packages/appeals-service-api/src/mappers/horizon-mapper.js

```diff
@@ -11,33 +10,41 @@ class HorizonMapper {
 	 * }
 	 */
 	appealToCreateOrganisationRequests(appeal) {
-		logger.debug('Constructing create organisation requests for Horizon')
-		let result = { appellant: { value: this.#getCreateContactRequestJson() } };
+		logger.debug('Constructing create organisation requests for Horizon');
+		let result = {
+			appellant: { value: this.#getCreateContactRequestJson('a:HorizonAPIOrganisation') }
+		};
+		result.appellant.value.AddContact.contact['a:Name'] = { '__i:nil': 'true' };
```

Comment:
this `['a:Name']` field could be added in the getCreateContactRequestJson function, as a condition if `type` == `'a:HorizonAPIOrganisation'`